dashboard/app.py

```diff
@@ -0,0 +1,318 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
+import plotly.express as px
+import seaborn as sns
+import matplotlib.pyplot as plt
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+from datetime import datetime
+import re
+import os
+import json  
+import itables.interactive
+from itables import show
+from IPython.core.display import display, display_html, Javascript, HTML
+
+import streamlit.components.v1 as components
+
+
+from pandas import json_normalize 
+
+
+DATA_URL = ('/mnt/ssardina-pacman/cosc1125-1127-AI/www/feedback-final/stats-archive')
+DEPLOYED_URL = 'http://118.138.246.177/feedback-final'
+ORGANIZER = 'RMIT AI - COSC1125/1127'
```

Comment:
This needs to be done better without hardcoding. If anything move it to a special file called `config.py`, maybe that is enough and document that that is the file to change to configure for a particular server

Reply:
I guess we can leave it like that by now and properly document it as I did in the README to tell the user to change it